packages/cypress-plugins/src/plugins/networkShim/reportMissingRequestStubs.js

```diff
@@ -0,0 +1,23 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+
+module.exports = function reportMissingRequestStubs(state, results) {
+    if (
+        results.totalFailed > 0 &&
+        Array.isArray(state.missingRequestStubs) &&
+        state.missingRequestStubs.length > 1
```

Comment:
Might be missing something, but should this be >=1 instead of >1 ?

Reply:
Shoot you are right, `> 0` would have made most sense. I'll prepare a fix PR and will merge that in cowboy style. Yeehaw 🤠 . (alpha is not protected)

Reply:
Could that be related to not being able to reproduce..?